src/mon/OSDMonitor.cc

```diff
@@ -2172,7 +2172,8 @@ void OSDMonitor::check_for_filestore_osds(health_check_map_t *checks)
     ss << filestore_osds.size()
        << " osd(s) "
        << (filestore_osds.size() == 1 ? "is" : "are")
-       << " on filestore; Filestore has been deprecated.";
+       << " on filestore; Filestore has been deprecated,"
+       << " the 'osd_op_queue' is enforced to 'wpq' for filestore OSDs.";
```

Comment:
How about: "...Filestore has been deprecated and the 'osd_op_queue' is enforced to 'wpq' for filestore OSDs."?

Reply:
nit: either use Filestore or filestore, let's keep it uniform